### ISSUE TITLE
Zły produkt w odpowiedzi

### ISSUE THREAD

**Issue by avello** [allegro/allegro-api#9730]
Występuje problem: 
 - [x] z zasobem REST API / with a REST API resource
 - [x] ze środowiskiem produkcyjnym / with the production environment

Pobieram dane produktu metodą GET /sale/products/627d357b-4b84-4c03-b103-da28453c3366
a w odpowiedzi dostaję produkt o ID 26033971-fdd5-47bd-bf67-37d7424fb8af

**Comment by MaciejFrackowiak** [comment]
Produkt został usunięty z naszej bazy, ponieważ wykryliśmy jego duplikat z id=26033971-fdd5-47bd-bf67-37d7424fb8af.

**Comment by avello** [comment]
No domyśliłem się, tylko czy w takim przypadku nie powinien być jakiś komunikat lub status 404

**Comment by MaciejFrackowiak** [comment]
Nie, ponieważ ten pierwotny produkt nie został całkowicie usunięty, a został połączony z produktem, który mieliśmy w bazie - dzięki tej operacji w ofercie nadal będzie widoczny produkt, co nie wygeneruje problemów w związku z jego brakiem.
